storybook/config/main.ts

```diff
@@ -1,5 +1,4 @@
 import type { StorybookConfig } from '@storybook/react-vite'
-import remarkGfm from 'remark-gfm'
```

Comment:
I now understand why we need this – it allows us to use GitHub Flavoured Markdown in Storybook.

For example, our tables no longer work without: https://amsterdam.github.io/design-system/demo-DES-618-storybook-8/?path=/docs/docs-design-guidelines-grid--docs#spacious-2

> This is a plugin to enable the extensions to markdown that GitHub adds with GFM: autolink literals, footnotes, strikethrough, tables, and tasklists. These extensions by GitHub to CommonMark are called [GFM](https://github.github.com/gfm/) (GitHub Flavored Markdown).

> Users frequently believe that some of these extensions, specifically autolink literals and tables, are part of normal markdown, so using remark-gfm will help match your implementation to their understanding of markdown.

– [source](https://github.com/remarkjs/remark-gfm)

Reply:
Thanks for the headsup, it works with the latest version.